packages/bierzo-wallet/src/logic/transactions/types/BwRegisterUsernameTx/ui/TransactionRow/Details.tsx

```diff
@@ -47,36 +41,57 @@ const TxDetails = ({ tx }: Props): JSX.Element => {
 
   return (
     <Block paddingLeft="56px" display="flex" flexDirection="column">
-      <Block margin={2} />
-      <Block>
-        <Typography variant="subtitle2" weight="regular" gutterBottom>
-          Personalized address:
-        </Typography>
-        <Typography
-          variant="subtitle2"
-          weight="regular"
-          color="textSecondary"
-          className={classes.sectionName}
-        >
-          {tx.original.username}
-        </Typography>
-        <Typography>&nbsp;</Typography>
-        <Table>
-          <TableHead>
-            <TableRow className={classes.tableHeader}>
-              <TableCell align="left">Blockchain</TableCell>
-              <TableCell align="left">Address</TableCell>
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {addresses.map(chain => (
-              <TableRow key={chain.chainId} className={classes.tableBody}>
-                <TableCell align="left">{chain.chainName}</TableCell>
-                <TableCell align="left">{chain.address}</TableCell>
-              </TableRow>
-            ))}
-          </TableBody>
-        </Table>
+      <Block display="flex">
+        <Block width="40%">
+          <Typography variant="subtitle2" weight="regular" gutterBottom>
+            Registered address:
+          </Typography>
+          <Typography
+            variant="subtitle2"
+            weight="regular"
+            color="textSecondary"
+            className={classes.sectionName}
+          >
+            {tx.original.username}
+          </Typography>
+        </Block>
+        <Block width="40%">
+          <Typography variant="subtitle2" weight="regular" gutterBottom>
+            Time:
+          </Typography>
+          <Typography variant="subtitle2" weight="regular" color="textSecondary">
+            {formatTime(tx.time)}
+          </Typography>
+        </Block>
+        <Block width="20%" paddingRight={3}>
+          <Typography variant="subtitle2" weight="regular" align="right" gutterBottom>
+            Transaction fee:
+          </Typography>
+          <Typography variant="subtitle2" weight="regular" color="textSecondary" align="right">
+            -{txFee}
```

Comment:
This will produce `--` in case if there is no fee. Or this is expected?

Reply:
You're right, it's because I wasn't expecting to have no fee for a Register Username Tx